index.bs

```diff
@@ -2207,6 +2214,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 :   <code><dfn for="assertionCreationData">signatureResult</dfn></code>
                 ::  whose value is the bytes of the signature value returned by the [=authenticator=].
 
+                :   <code><dfn for="assertionCreationData">authenticatorAttachmentResult</dfn></code>
```

Comment:
I don't think this needs to be part of `assertionCreationData` since the attachment doesn't come from the authenticator. Move this section directly to the bit below where it gets added to `PublicKeyCredentialResponse` (in this patch, that's line 2256).